app/src/main/java/org/simple/clinic/facilitypicker/FacilityPickerUi.kt

```diff
@@ -5,7 +5,5 @@ import org.simple.clinic.facility.change.FacilityListItem
 interface FacilityPickerUi {
   fun showProgressIndicator()
   fun hideProgressIndicator()
-  fun showToolbarWithSearchField()
-  fun showToolbarWithoutSearchField()
   fun updateFacilities(facilityItems: List<FacilityListItem>)
 }
```

Comment:
Can you tell me how is switching toolbar type related to progress indicator or shimmer effect in this PR?

Reply:
Previously we had 2 different toolbars. One is with search bar and another one is standard toolbar. Since in facility picker we always need search bar I have removed the standard toolbar and then used the progress indicator for search toolbar.

https://simpledotorg.slack.com/archives/CFK2PHJ1L/p1612939938210800